RELEASES.md

```diff
@@ -635,3 +635,9 @@
 10.2.1 2019-08-16
 -----------------
   - Issue: access denied security logs username and password via JSON.stringify
+
+10.2.2 2019-08-16
+-----------------
+  - session logs browser primus writer
+  - revokeSession revokes token, including across the cluster
+  - reverted to v6 of ws
```

Comment:
i know you said this is for the shoreline issue - but can you please add reasoning to the history file on why it needs to be reverted

Reply:
comment will be: "reverted to v6 of ws - backward compatibility issue"
as I dont think documenting "shoreline" in this open source repo makes sense - you ok with that?

Reply:
would be useful to just state what the backwards compatibility issue is if you can elaborate some more - if we need to update this in the future again it should provide enough background on what the issue was on why we couldn't use that version